app/views/about/index.html.erb

```diff
@@ -140,5 +140,6 @@
       <p id="copyright-p"><%= (_("Version %{version}  %{copyright} 2009-%{year} Paul Kelly and %{author}") % {:version => SETTINGS[:version], :copyright => "&copy;", :year=>DateTime.now.year, :author=>mail_to("ohadlevy@gmail.com", "Ohad Levy" )}).html_safe %></p>
     </div>
   </div>
+  <%= slot('aboutFooterSlot', true) %>
 </div>
-</div>
```

Comment:
Why did you remove the closing div tag? Is this aim to fix something different than the scope of this PR? Or did it happen by accident?

Reply:
this is an extra closing div (with no starting one)